app/core/config.py

```diff
@@ -25,7 +25,7 @@
 from .logging import InterceptHandler
 
 VERSION = "0.0.0"
-API_PREFIX = "/"
+API_PREFIX = "/api"
```

Comment:
What is the full url for the resolver going to be?

Reply:
Make API_PREFIX empty